src/components/Dropdown/asyncDropdown/AsyncDropdown.jsx

```diff
@@ -76,6 +82,15 @@ class AsyncDropDown extends React.Component {
           }))
         : [],
       loading: false,
+      keepOpen: this.state.keepOpen || !!query,
```

Comment:
du kan gjøre som på linke 65 for å unngå å skrive this.state men det er ikke så farlig, bare en estetisk preferanse 💅